projects/step-frontend/src/lib/modules/timeseries/performance-view/filter-bar/filter-bar.component.ts

```diff
@@ -246,4 +261,35 @@ export class FilterBarComponent implements OnInit, OnDestroy {
     }
     return false;
   }
+
+  exportRawData() {
+    if (this.exportInProgress) {
+      return;
+    }
```

Comment:
it's a good practice in terms of clean code to leave an empty line before/after if/for/etc.